src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -481,6 +503,11 @@ void manualRegistrationToolBox::disableSaveButtons(bool param)
     d->b_exportTransformation->setDisabled(param);
 }
 
+void manualRegistrationToolBox::disableComputeButton(bool param)
+{
+    d->b_computeRegistration->setDisabled(param);
+}
```

Comment:
Is this method really useful?
i.e will a dev willing to enable/disable this button look for a method in the tbx to do that? or will s/he just call directly the button's method?